graph-db/serverless/src/indexCmrCollection/__test__/handler.test.js

```diff
@@ -120,6 +120,28 @@ describe('indexCmrCollection handler', () => {
     expect(statusCode).toBe(200)
   })
 
+  test('test indexing of a broken collection', async () => {
+    // Pass an error to the response of fetchCmrCollection
+    nock(/local-cmr/)
```

Comment:
do you mean to involve regular expressions? Is this a requirement of the function?